.codecov.yml

```diff
@@ -1,17 +1,1140 @@
 comment:
-  layout: "files"
+  layout: files
   behavior: default
   branches: null
-
-
 coverage:
   range: 50..100
   round: down
   precision: 2
-
   status:
     project:
       default:
         target: 75
         if_ci_failed: error
-    patch: off
+      .NET CLR:
```

Comment:
Are all these manual entries really necessary?  Seems like yet another thing to keep in sync ..

Reply:
Yes

> Seems like yet another thing to keep in sync

I don't see an issue with that

Reply:
Ok, I see its a requirement of the codecov format - extra verbose to have to repeat so much but 🤷